### ISSUE TITLE
Why the versionName is 2.0.0?

### ISSUE THREAD

**Issue by abdonrd** [GoogleChromeLabs/llama-pack#68]
Why the versionName is 2.0.0?

https://github.com/GoogleChromeLabs/llama-pack/blob/2137ae160ffa9c64dfbe97e04815185ed733a66d/template_project/app/build.gradle#L47

There are any way to manage it? Or should we edit it manually from there?

**Comment by andreban** [comment]
That's a left-over from [svgomg-twa]((https://github.com/GoogleChromeLabs/svgomg-twa), which was used as a base for the template. We can reset that back to `1.0.0`.

Nevertheless, we need a mechanism to enable #3, where `versionCode` and `versionName` are updated with the template. 

One option is to control this via the template. We'd have to bump `versionName` and `versionCode` on the template when it's changed, and `update` would apply the new template.

The second option (somewhat my preferred, is to save the current versionCode and versionName to `twa-manifest.json` and bump it when `llama-pack update` is invoked. We'd still probably want to have a way to track template versions, for debugging purposes.

**Comment by abdonrd** [comment]
Reading about [Versioning Android apps here](https://medium.com/@maxirosson/versioning-android-apps-d6ec171cfd82)...

What do you think of having a `version` field in the manifest?
And let the `build.gradle` generate `versionCode` and `versionName`.

Something like:

```json
...
"version": "2.3.1",
...
```

And then the `build.gradle` generates:

```gradle
versionCode 020301
versionName "2.3.1"
```

**Comment by andreban** [comment]
`versionCode` is the bit that really matters on the Play Store, and the number must always go up between versions. I think we can just have the 2 values on the twa-manifest and update them independently. eg: 

```json
...
"version": {
    "code": 1,
    "name": "1.0.0"
} 
...
```

**Comment by abdonrd** [comment]
Yes, but following a formula like this, the `versionCode` would always increase:

```javascript
const versionCode = (majorVersion * 10000) + (minorVersion * 100) + patchVersion;
```

So a version like `"2.3.1"` generates the version code `20301`.

But having both options also works. 👍

**Comment by andreban** [comment]
Moving the conversation on how to update versions to #74 and closing this issue, as resetting the `versionName` has been implemented on #71.
